src/peergos/server/Command.java

```diff
@@ -60,7 +62,7 @@ public void main(Args args) {
         if (headOpt.isPresent()) {
             String head = headOpt.get();
             if (head.equals("help")) {
-                System.out.println(helpMessage());
+                LOG.info(helpMessage());
```

Comment:
I think the idea with the cmd interface is you print exactly what you want, so we wouldn't want the logger here.

Reply:
same with all the other commands